### PR TITLE
[pulsar-client-tools] Add support for websocket produce/consume command

### PR DESCRIPTION
**Pull request by rdhabalia**
### Motivation
Adding websocket client support into cli produce/consume command.

### Modification
Add ws produce/consume in cli-tool.

### Result
Now, user will be able to produce and consume messages to websocket proxy by just defining broker-service url as `ws://<host>:<port>`.
eg:
```
consume: 
./pulsar-client  --url ws://localhost:8080 consume  persistent://sample/standalone/ns1/t1 -s s1 -n 100

produce:
./pulsar-client  --url ws://localhost:8080/ produce -m "test" -n 100 persistent://sample/standalone/ns1/t1 
```

### CODE REVIEW

**Comment by rdhabalia** [comment]
rerun java8 tests
rerun integration tests

**Comment by rdhabalia** [comment]
rerun integration tests

File: pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java [hunk 1]
```diff
@@ -151,5 +188,152 @@ public int run() throws PulsarClientException, IOException {
...
+
+        TopicName topicName = TopicName.get(topic);
+        
+        String wsTopic = String.format(
+                "%s/%s/" + (StringUtils.isEmpty(topicName.getCluster()) ? "" : topicName.getCluster() + "/")
+                        + "%s/%s/%s?subscriptionType=%s",
+                topicName.getDomain(), topicName.getTenant(), topicName.getNamespacePortion(), topicName.getLocalName(),
+                subscriptionName, subscriptionType.toString());
+        
+        /*String wsTopic = topicName.getDomain() + "/" +  topicName.getTenant()
```
Comment (jiazhai) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java]: remove this？

Comment (rdhabalia) [reply pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java]: yes..fixed it.

**Comment by sijie** [comment]
run java8 tests

**Pull request merged**